### PR TITLE
Follow up of #2564

### PR DESCRIPTION
**Pull request by meg-gupta**

### CODE REVIEW

File: CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift [hunk 1]
```diff
@@ -586,11 +561,12 @@ public let TYPE_NODES: [Node] = [
       Child(
         name: "scopedKeyword",
         kind: .token(choices: [.keyword(.scoped)]),
+        documentation: "scoped lifetime dependence specifier",
```
Comment (ahoppen) [inline CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift:564]: Could you be a little more elaborate here, ie. what `scoped` actually does.

File: CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift [hunk 2]
```diff
@@ -542,42 +542,17 @@ public let TYPE_NODES: [Node] = [
...
-  ),
-
   Node(
     kind: .lifetimeTypeSpecifier,
     base: .syntax,
     experimentalFeature: .nonescapableTypes,
     nameForDiagnostics: "lifetime specifier",
     documentation: "A specifier that specifies function parameter on whose lifetime a type depends",
-    traits: [
-      "Parenthesized"
```
Comment (ahoppen) [inline CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift:574]: I think you need to update some `CodeGeneration` test for this. Could you run the tests in the `CodeGeneration` module?

Comment (meg-gupta) [reply CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift:574]: Done.

File: CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift (hunk 1 above)
Comment (meg-gupta) [reply CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift]: Updated to : lifetime of return value is scoped to the lifetime of the original value

**Comment by meg-gupta** [comment]
@swift-ci test

**Review by ahoppen — APPROVED** [review]

**Comment by meg-gupta** [comment]
@swift-ci test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**